test/unit/test_buf.c

```diff
@@ -1055,6 +1055,32 @@ bool test_rz_buf_get_string_nothing(void) {
 	mu_end;
 }
 
+bool test_rz_buf_get_nstring(void) {
+	ut8 *ch = malloc(128);
+	memset(ch, 'A', 127);
+	ch[127] = '\0';
+	ch[111] = '\0';
+	RzBuffer *b = rz_buf_new_with_bytes(ch, 128);
+	char *s = rz_buf_get_nstring(b, 100, 10);
+	mu_assert_null(s, "there is no string with size < 10 (no null terminator)");
+	free(s);
+	s = rz_buf_get_nstring(b, 100, 11);
+	mu_assert_streq(s, (char *)ch + 100, "the string is the same");
+	free(s);
```

Comment:
```suggestion
	mu_assert_streq_free(s, (char *)ch + 100, "the string is the same");
```
You can do this if you want